build/charts/antrea/templates/antctl/clusterrole.yaml

```diff
@@ -26,9 +26,12 @@ rules:
       - networkpolicystats
       - antreaclusternetworkpolicystats
       - antreanetworkpolicystats
+      - nodeiplatencystats
```

Comment:
I am fine with this change, even though it's not required by this PR, because AFAIK we are not adding antctl support.

Reply:
OK, I was testing earlier and finded that the `antrea agent` needs to configure permissions here when it sends a `create` request by provider, so I added it.